pkg/sensors/tracing/generickprobe.go

```diff
@@ -211,6 +216,15 @@ func createMultiKprobeSensor(sensorPath string, multiIDs, multiRetIDs []idtable.
 	var progs []*program.Program
 	var maps []*program.Map
 
+	oneKprobeHasStackTrace := false
+	for _, id := range multiIDs {
+		gk, err := genericKprobeTableGet(id)
+		if err != nil {
+			continue
```

Comment:
I had to write this because we haven't https://github.com/cilium/tetragon/pull/2145, not sure it's okay to ignore that error like that, we could end up with a kprobe with stacktrace that we fail to retrieve from the kprobe table and then we don't resize the map properly. Might be difficult to debug that the issue is that the map is of size 1. wdyt @olsajiri ?

Reply:
I think we should fail like we do in upstream

```
        for _, id := range multiIDs {
                gk, err := genericKprobeTableGet(id)
                if err != nil { 
                        return nil, nil, err
                }
```

Reply:
yes I would love to as well but the signature of the function in v1.0:
```go
func createMultiKprobeSensor(sensorPath string, multiIDs, multiRetIDs []idtable.EntryID) ([]*program.Program, []*program.Map)
```
Should I backport those changes as well then? There's no error returned in this version